gstcefsrc.cc

```diff
@@ -228,17 +228,36 @@ class AudioHandler : public CefAudioHandler
     IMPLEMENT_REFCOUNTING(AudioHandler);
 };
 
+class DisplayHandler : public CefDisplayHandler {
+public:
+  DisplayHandler(GstCefSrc *element) : mElement(element) {}
+
+  ~DisplayHandler() = default;
+
+  virtual bool OnConsoleMessage(CefRefPtr<CefBrowser>, cef_log_severity_t level,
+                                const CefString &message,
+                                const CefString &source, int line) override {
+    GST_DEBUG_OBJECT (mElement, "%s:%d : %s", source.ToString().c_str(), line, message.ToString().c_str());
```

Comment:
Ah depending on the level notified we could use GST_INFO or other gst logging level.